packages/vscode-ui5-language-assistant/scripts/package-vsix.js

```diff
@@ -59,7 +59,10 @@ const { packageCommand } = proxyquire("vsce/out/package", {
 });
 
 const pkgJsonPath = resolve(rootPkgDir, "package.json");
-const pkgJsonOrg = readJsonSync(pkgJsonPath);
+// Read & save the original literal representation of the pkg.json
+// To avoid dealing with re-formatting (prettier) later on.
+const pkgJsonOrgStr = readFileSync(pkgJsonPath, "utf8");
+const pkgJsonOrg = JSON.parse(pkgJsonOrgStr);
```

Comment:
Now you don't need to `cloneDeep` before changing the `main`

Reply:
ah thanks